docs/general/ad-filtering/create-own-filters.md

```diff
@@ -204,6 +204,59 @@ AdGuard Safari and AdGuard for iOS do not fully support regular expressions beca
 
 :::
 
+### Restrictions on rules application {#rules-restrictions}
+
+Rules that match an arbitrarily large number of URLs are considered incorrect and will be ignored. This can happen if the rule doesn't contain a mask, or if the mask matches any URL with a certain protocol.  
```

Comment:
```suggestion
Rules that match an arbitrarily large number of URLs are considered incorrect and will be ignored. This can happen if the rule doesn't contain a mask, or if the mask matches any URL with a certain protocol.
```